server/models/models.go

```diff
@@ -50,6 +50,7 @@ type Session struct {
 	Token      string
 	Username   string
 	Expiration time.Time
+	RememberMe string
```

Comment:
Is this a string and not a bool because javascript passes it as a string?

If so, can we change this to a bool and set it based on what we get back? IE:

`RememberMe = param == "true" ? true : false`